R/stage_nldas_ts.R

```diff
@@ -16,24 +16,24 @@
 #'   
 #' @examples
 #' \dontrun{
-#' files <- stage_nldas_ts(sites = c("nwis_06893820","nwis_01484680"), variable = "baro", 
+#' files <- stage_nldas_ts(sites = c("nwis_06893820","nwis_01484680"), var = "baro", 
 #'   times = c('2014-01-01 00:00','2014-01-01 05:00'), verbose = TRUE)
 #' read_ts(files[1])
 #' }
 #' @export
-stage_nldas_ts <- function(sites, variable, times, folder = tempdir(), verbose = FALSE, ...){
+stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALSE, ...){
   
-  if (length(variable) > 1) 
-    stop ('variable must be single value.')
+  if (length(var) > 1) 
+    stop ('var must be single value.')
   
   # get site coordinates and format for geoknife
   lon_lat <- find_site_coords(sites, format="geoknife")
   lon_lat_df <- lon_lat[complete.cases(t(lon_lat))]
   
-  p_code <- get_var_codes(variable, "p_code")
+  p_code <- get_var_codes(var, "p_code")
   
   stencil <- simplegeom(lon_lat_df)
-  fabric <- webdata('nldas', variables = p_code, times = times)
+  fabric <- webdata('nldas', variable = p_code, times = times)
```

Comment:
interesting. ?geoknife::webdata has a slot named `variable` and an example that uses `variables`, so i'm not actually  sure which one it requires. i might have changed that due to a R CMD check warning...haven't tested this function much yet. it's next after i finish working with stage_nwis_ts + makefile.